libpod/runtime_ctr.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const ctrRemoveTimeout = 10
```

Comment:
I think this is fine, but am going to ask just to be 100% sure, should this be in a conf file so it could be adjusted?  I don't think so, but I'm not positive.

Reply:
I think for now, I want to avoid a config file.